Source/MonoGame.Extended/Timers/ContinuousClock.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MonoGame.Extended.Timers
+{
+    public class ContinuousClock : IGameComponent
+    {
+        public event EventHandler ReportTimePassed;
+
+        public TimerState TimerState;
+        public double ReportEveryXSeconds;
+
+        public TimeSpan CurrentTime { get; protected set; } = TimeSpan.Zero;
+        public TimeSpan NextReport { get; protected set; } = TimeSpan.Zero;
```

Comment:
Alright, wasn't sure about that, though decided to use it because I've been trying to get used to the new C# 6.0 stuf